tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md

```diff
@@ -0,0 +1,131 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product-function>ui-sap-business-client--nwbc-]
+primary_tag: software-product-function>ui-sap-business-client--nwbc-
+author_name: Sandra Thimme
+author_profile: https://github.com/sandra36
+---
+
+# Launch native SAP GUI for Windows Applications in the Cloud with SAP Business Client
+<!-- description --> Running SAP Build Work Zone, standard edition inside SAP Business Client you can mix native SAP GUI transactions with state-of-the-art web applications. Therefore, a TCP ping to the remote system has to be performed at the time the application shall be started.
+
+## You will learn
```

Comment:
Don't ask questions in this sections. Just tell the user what he will learn. Please check the style guide for more information.

Reply:
Please revise like the following:
What SAP Build Work Zone is --> "the definition of SAP Build Work Zone"
What the Prerequisites for the integration of SAP Build Work Zone are --> "the prerequisites for the integration of SAP Build Work Zone"
What the difference between direct and tunneled Access is --> "the difference between direct and tunneled Access"
How to run a cloud launchpad within SAP Business Client in 3 steps.